### PR TITLE
feat: getting started page enhancements

### LINKED ISSUE #1142: Getting started page enhancements
These are a couple of enhancements that need to be added to the Getting Started Page

Redlines: https://www.figma.com/file/KZ6SWR2abIniIJY4RewDeA/RL_Designing_GetStarted?node-id=10%3A13436
Updates:  https://ibm.ent.box.com/notes/997577728498?s=37v97opzd2ea21ffaaepqlyynhiybkq4

- [x] Make sure all kits are added according to figma
- [x] IBM Brand needs to be sorted first 
~Make sure type: elements render first~
- [x] Add wireframe design kits to Sketch tab
- [x] All items in the table should either be links (to figma design kit previews) or downloads of design kits. This should all be defined in the schema.
- [x] Figma tab is missing elements design kits
- [x] Axure is missing `widget` kit
- [x] Move IBM Grid - IBM Brand

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #1142 

- Figma:  https://www.figma.com/file/KZ6SWR2abIniIJY4RewDeA/RL_Designing_GetStarted?node-id=10%3A13436
- Box Notes for specific changes: https://ibm.ent.box.com/notes/997577728498?s=37v97opzd2ea21ffaaepqlyynhiybkq4

## Testing:  

- Make sure functionality and designs follow Figma above
- http://localhost:3000/designing/get-started

### CODE REVIEW

**Comment by alisonjoseph** [comment]
Loks like the table and dropdown don't hang left in the Figma file, but the coded table does hang. I also double checked the assets table and it hangs left too. Wasn't sure what was correct? If you match Figma file, should we also update the Assets table to not hang?

Screen Shot 2022-08-18 at 3 36 02 PM

**Comment by jeanservaas** [comment]
Yeah, looks like there's some confusion about hanging and not hanging tables... and I can see why. Since the catalog is a custom table and it hangs and It has the same sort dropdown as our data tables do (which is eventually going to become a fluid input) — it probably makes sense to also hang the data tables. But I'll leave this up to @kingtraceyj 

Even if we decide not to hang the data tables, we just need to make sure then that none of them are hanging.

image

**Comment by kingtraceyj** [comment]
@alisonjoseph talking to @jeanservaas we will go ahead a hang the table since we will have fluid drop downs at some point. i'll update the redlines.

File: packages/resources/carbon.yml [hunk 1]
```diff
@@ -283,3 +290,59 @@ designKits:
...
+    type: ui
+    status: stable
+    url: https://www.figma.com/file/YAnB1jKx0yCUL29j6uSLpg/(v11-BETA)-White-Theme---Carbon-Design-System
+    action: link
+  figma-v11-gray-10-theme:
+    name: Figma v11 gray 10 theme
+    tool: figma
+    type: ui
+    status: stable
+    url: https://www.figma.com/file/fH8rkv1VUTw5pvFDvO1kPO/(v11-BETA)-Gray-10-Theme---Carbon-Design-System
```
Comment (aubrey-oneal) [inline packages/resources/carbon.yml:305]: For some reason this link doesn't work for me-- it switches to the gray 90 theme upon loading the link. Try this link instead:

https://www.figma.com/file/0sd447WTcazDI6Dq9tjXOe/(v11-BETA)-Gray-10-Theme---Carbon-Design-System?node-id=58%3A2763

Comment (andreancardona) [reply packages/resources/carbon.yml:305]: we had the wrong link - fixed it!

**Review by alisonjoseph — APPROVED** [review]

File: services/web-app/components/filterable-design-kit-table/filterable-design-kit-table.js [hunk 2]
```diff
@@ -100,16 +110,20 @@ const captions = {
 }
 
 const FilterableDesignKitTable = ({ designKitsData, designTools, designKitIds }) => {
+  const displayDropdown = designTools?.length > 0
   const [filteredRows, setFilteredRows] = useState(designKitsData)
-  const [currentItem, setCurrentItem] = useState(designTools[0])
+  const [currentItem, setCurrentItem] = useState(displayDropdown && designTools[0])
```
Comment (francinelucca) [inline services/web-app/components/filterable-design-kit-table/filterable-design-kit-table.js:115]:
```suggestion
  const [currentItem, setCurrentItem] = useState(displayDropdown ? designTools[0] : undefined)
```

File: services/web-app/components/filterable-design-kit-table/filterable-design-kit-table.module.scss [hunk 3]
```diff
@@ -18,3 +20,31 @@
     }
   }
 }
+
+:global(.cds--data-table) tbody {
```
Comment (francinelucca) [inline services/web-app/components/filterable-design-kit-table/filterable-design-kit-table.module.scss:24]:
```suggestion
// override carbon data table body
:global(.cds--data-table) tbody {
```

File: services/web-app/pages/designing/get-started.mdx [hunk 4]
```diff
@@ -168,18 +165,28 @@ design kits include elements, core components, wireframes, and guidelines.
...
+    'v11-text-styles-figma',
+    'color-styles-figma'
   ]}
 />
 
 ### Local design kits
 
 Additional design kits compliment and extend Carbon. For example, IBM.com provides components for an
-editorial approach. More kits can be found in the [design kit catalog]().
+editorial approach. More kits can be found in the [design kit catalog](). The kits in the table
```
Comment (francinelucca) [inline services/web-app/pages/designing/get-started.mdx:185]:
```suggestion
<!-- TODO: add design kit catalog url -->
editorial approach. More kits can be found in the [design kit catalog](). The kits in the table
```

**Review by francinelucca — APPROVED** [review]
LGTM, minor suggestions

**Pull request merged**